src/common/routes.js

```diff
@@ -14,5 +15,6 @@ export default (
     <Route path="/:owner/:name/builds" component={Builds}/>
     <Route path="/:owner/:name/builds/:buildId" component={BuildDetails}/>
     <Route path="/login/failed" component={LoginFailed}/>
+    <Route path="/dashboard/select-repositories" component={SelectRepositories}/>
```

Comment:
FWIW `dashboard` is a reserved GitHub username (it must be, since https://github.com/dashboard has a special meaning), so if we want to use that at our top level it's safe to do so.  `settings` is similarly reserved.